### ISSUE TITLE
Add record timestamp when event is decorated

### ISSUE THREAD

**Issue by consulthys** [logstash-plugins/logstash-input-kafka#176]
Since Kafka 0.10.0.0, it is now possible to [include a timestamp in Kafka records](https://cwiki.apache.org/confluence/display/KAFKA/KIP-32+-+Add+timestamps+to+Kafka+message).

It would be great if it was possible to also retrieve the timestamp in the kafka input plugin [when the event is decorated](https://github.com/logstash-plugins/logstash-input-kafka/blob/master/lib/logstash/inputs/kafka.rb#L254-L259).

Basically, this should only be a one-liner change in the linked code:
```
event.set("[kafka][timestamp]", record.timestamp)
```

Related issue: https://github.com/logstash-plugins/logstash-output-kafka/issues/124

**Comment by consulthys** [comment]
@suyograo this would also be consistent with the work you're doing in https://github.com/logstash-plugins/logstash-input-kafka/tree/feature/seek_by_time
